plugin/trino-cassandra/src/main/java/io/trino/plugin/cassandra/CassandraClusteringPredicatesExtractor.java

```diff
@@ -48,15 +48,14 @@ public String getClusteringKeyPredicates()
 
     public TupleDomain<ColumnHandle> getUnenforcedConstraints()
     {
-        Map<ColumnHandle, Domain> pushedDown = clusteringPushDownResult.getDomains();
-        return predicates.filter(((columnHandle, domain) -> !pushedDown.containsKey(columnHandle)));
+        return predicates.filter(((columnHandle, domain) -> clusteringPushDownResult.hasNotBeenFullyPushed(columnHandle)));
     }
 
     private static ClusteringPushDownResult getClusteringKeysSet(List<CassandraColumnHandle> clusteringColumns, TupleDomain<ColumnHandle> predicates, VersionNumber cassandraVersion)
     {
-        ImmutableMap.Builder<ColumnHandle, Domain> domainsBuilder = ImmutableMap.builder();
+        ImmutableMap.Builder<ColumnHandle, Domain> fullyPushedDomains = ImmutableMap.builder();
```

Comment:
`fullyPushedDomains` is a Map, but its values are apparently never read. did you mean to make it a Set?

Reply:
So I've left it as a map, as that's how it has been implemented before (although it's values weren't used), I've just changed the names a bit to better reflect what's going on here. It could be a Set though. I can change it to one.

Reply:
> So I've left it as a map, as that's how it has been implemented before (although it's values weren't used)

could be nice prep commit

> I've just changed the names a bit to better reflect what's going on here

that's a good change in general

as it is a Map, i though we're comparing fully pushed down domains with some other domains, so was a bit confused when realized this isn't the case